packages/plugin-msdynamic-ui/src/containers/ExchangeRateSettings.tsx

```
@@ -0,0 +1,67 @@
+import { gql } from '@apollo/client';
+import * as compose from 'lodash.flowright';
+import { graphql } from '@apollo/client/react/hoc';
+import { Alert, withProps } from '@erxes/ui/src/utils';
+import List from '../components/settings/ExchangeRateSettings';
+import { ConfigsQueryResponse, IConfigsMap } from '../types';
+import { mutations, queries } from '../graphql';
+import React from 'react';
+import Spinner from '@erxes/ui/src/components/Spinner';
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)